libshaderc/include/shaderc/shaderc.hpp

```diff
@@ -55,25 +72,30 @@ class CompilationResult {
     return shaderc_result_get_compilation_status(compilation_result_);
   }
 
-  // Returns a pointer to the start of the compiled SPIR-V.
-  // It is guaranteed that static_cast<uint32_t> is valid to call on this
-  // pointer.
-  // This pointer remains valid only until the CompilationResult is destroyed.
-  const char* GetData() const {
-    if (!compilation_result_) {
-      return "";
-    }
-    return shaderc_result_get_bytes(compilation_result_);
+  // Returns an iterator pointing to the start of the compilation
```

Comment:
There's a comment at the top of the class describing the kinds of iterators being handed out.  But next CL will expand this comment to say
"Returns a random access (contiguous) iterator ..."